Tools/autotest/sim_vehicle.py

```diff
@@ -731,10 +818,10 @@ def start_mavproxy(opts, stuff):
                "you are simulating, for example, start in the ArduPlane directory to " \
                "simulate ArduPlane")
 
-parser.add_option("-v", "--vehicle", type='string', default=None, help="vehicle type (ArduPlane, ArduCopter, APMrover2 or ArduSub)")
-parser.add_option("-f", "--frame", type='string', default=None, help="""set aircraft frame type
-                     for copters can choose +, X, quad or octa
-                     for planes can choose elevon or vtail""")
+parser.add_option("-v", "--vehicle", type='choice', default=None, help="vehicle type %s" % vehicle_options_string, choices=_options.keys())
```

Comment:
Surprisingly, no.  There might be a shortcut to getting the options string, 'though.